torch_geometric/explain/utils/metrics.py

```diff
@@ -0,0 +1,63 @@
+import torch
+from torch import Tensor
+
+
+def get_characterization_score(weights: Tensor,
+                               fidelities: Tensor) -> 'Tensor':
+    r"""Returns the componentwise characterization score of
+    fidelities[0] and fidelities[1]
+
+
+    Args:
+        weights (Tensor): Tensor containing the two weights
```

Comment:
would be nice to include the formula for characterization and say what the weights correspond to?
Also we should have a dimension for weights. I'm ok with a pair of 2 numbers? It's ok as long as we document it.

Reply:
Added both, still unsure the latex compiles directly as I'm unable to get sphinx to build a link to the docs (require some help here for this)